File: lang/go/src/fun/vac/algorithms/meta/Sorting/Sorting.go
package Sorting

import (
    bubble_sort    "fun/vac/algorithms/sorting/BubbleSort"
    insertion_sort "fun/vac/algorithms/sorting/InsertionSort"
    merge_sort     "fun/vac/algorithms/sorting/MergeSort"
    quick_sort     "fun/vac/algorithms/sorting/QuickSort"
    selection_sort "fun/vac/algorithms/sorting/SelectionSort"
)

func BubbleSort(arr []int) {
    bubble_sort.Sort(arr)
}

func InsertionSort(arr []int) {
    insertion_sort.Sort(arr)
}

func MergeSort(arr []int) {
    merge_sort.Sort(arr)
}

func QuickSort(arr []int) {
    quick_sort.Sort(arr)
}

func SelectionSort(arr []int) {
    selection_sort.Sort(arr)
}

File: lang/nodejs/src/fun/vac/algorithms/meta/Sorting.js
"use strict"

const BubbleSort    = require("fun/vac/algorithms/sorting/BubbleSort")
const InsertionSort = require("fun/vac/algorithms/sorting/InsertionSort")
const MergeSort     = require("fun/vac/algorithms/sorting/MergeSort")
const QuickSort     = require("fun/vac/algorithms/sorting/QuickSort")
const SelectionSort = require("fun/vac/algorithms/sorting/SelectionSort")

exports.bubbleSort = (arr) => {
    BubbleSort.sort(arr)
}

exports.insertionSort = (arr) => {
    InsertionSort.sort(arr)
}

exports.mergeSort = (arr) => {
    MergeSort.sort(arr)
}

exports.quickSort = (arr) => {
    QuickSort.sort(arr)
}

exports.selectionSort = (arr) => {
    SelectionSort.sort(arr)
}
